web_ui/dev-proxy.ts

```diff
@@ -66,13 +66,19 @@ export const devProxy: ProxyConfig = [
                 return buffer;
             }
 
-            const originalFeatureFlags = JSON.parse(buffer.toString('utf8'));
+            const content = buffer.toString('utf8');
```

Comment:
why this change (the whole file)? If needed, could we make it on a different PR?, seems a bit out of scope of this one :P

Reply:
Moved to #1217